apps/admin-portal/src/api/users.ts

```diff
@@ -44,3 +49,76 @@ export const getUsersList = (): Promise<any> => {
             return Promise.reject(error);
         });
 };
+
+/**
+ * Retrieve the list of user stores that are currently in the system.
+ *
+ * @returns {Promise<BasicProfileInterface>} a promise containing the user store list.
+ */
+export const getUserStoreList = (): Promise<any> => {
+    const requestConfig = {
+        headers: {
+            "Access-Control-Allow-Origin": GlobalConfig.clientHost,
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.GET,
+        url: ServiceResourcesEndpoint.userStores
+    };
+
+    return httpClient.request(requestConfig)
+        .then((response) => {
+            return Promise.resolve(response);
+        })
+        .catch((error) => {
+            return Promise.reject(error);
+        });
+};
+
+/**
+ * Add new user.
+ *
+ * @returns {Promise<BasicProfileInterface>} a promise containing the response.
+ */
+export const addUser = (data: object): Promise<any> => {
```

Comment:
@param description missing in the doc comment. Please check the other places as well.